index.html

```diff
@@ -0,0 +1,30 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" text="text/css" href="css/my_website.css">
+    <title>Alejandro Ortiz</title>
+</head>
+
+<body>
+    <div class="main-header">
+        <h1 id="title">Alejandro Ortiz</h1>
+    </div>
+    <p class="text">Hi, I’m @alejandro-ortiz <br>
+        I’m interested in Rocketry, Tennis, Web Development, Electronics, and scouts<br>
+        I’m currently learning Git and Javascript
+    </p>
+    <div class="text2">
```

Comment:
The names of these classes should be more meaningful. What does the name of this class communicate, and how would someone know when to use text, text2 or text3?

Reply:
I simplified that by having one called paragraph and removing the rest